kubernetes-schema.sql

```diff
@@ -40,12 +49,4 @@ CREATE TABLE "Service" (
   "port" int
 );
 
-ALTER TABLE "Node" ADD FOREIGN KEY ("cluster_id") REFERENCES "Cluster" ("cluster_id");
-
-ALTER TABLE "Pod" ADD FOREIGN KEY ("cluster_id") REFERENCES "Cluster" ("cluster_id");
-
-ALTER TABLE "Pod" ADD FOREIGN KEY ("node_id") REFERENCES "Node" ("node_id");
-
-ALTER TABLE "Container" ADD FOREIGN KEY ("pod_id") REFERENCES "Pod" ("pod_id");
-
 ALTER TABLE "Service" ADD FOREIGN KEY ("cluster_id") REFERENCES "Cluster" ("cluster_id");
```

Comment:
Why does this constrain remain?

Reply:
Because I did not alter the Service table, as I did not see a reason to include a history there currently.